recipes/cxxtest/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "cxxtest" %}
+{% set version = "4.4" %}
+{% set sha256 = "1c154fef91c65dbf1cd4519af7ade70a61d85a923b6e0c0b007dc7f4895cf7d8" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+     
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    url: https://github.com/CxxTest/cxxtest/releases/download/4.4/cxxtest-4.4.tar.gz
```

Comment:
Please template the version using `{{ version }}` in the URL.

Reply:
Done